docs/oscar_references.bib

```diff
@@ -36,6 +36,21 @@ @InProceedings{BDLP19
   pdf           = {https://hal.inria.fr/hal-02912148/file/07.pdf}
 }
 
+@inproceedings{BES21,
+    author 			= {Berthomieu, J\'{e}r\'{e}my and Eder, Christian and Safey El Din, Mohab},
+    title 			= {Msolve: A Library for Solving Polynomial Systems},
+    year 				= {2021},
+    publisher 	= {Association for Computing Machinery},
+    address 		= {New York, NY, USA},
+    url 				= {https://doi.org/10.1145/3452143.3465545},
+    doi 				= {10.1145/3452143.3465545},
+    booktitle 	= {Proceedings of the 2021 on International Symposium on Symbolic and Algebraic Computation},
+    pages 			= {51-–58},
+    keywords 		= {multivariate polynomial systems solver, c library, zero-dimensional systems},
+    location 		= {Virtual Event, Russian Federation},
+    series 			= {ISSAC '21}
+    }
```

Comment:
Could you perhaps follow the instructions in `docs/README.md` and run `bibtool docs/oscar_references.bib -o docs/oscar_references.bib` to format the bib consistently?

(This is of course not a blocker, we can also do it later, e.g. in case installing `bibtool` is too much of a hassle)